lib/Dialect/HW/HWOps.cpp

```diff
@@ -1258,7 +1258,10 @@ LogicalResult InstanceOp::verify() {
   for (auto param : parameters()) {
     auto paramAttr = param.cast<ParamDeclAttr>();
     auto value = paramAttr.getValue();
-    assert(value && "SymbolUses verifier should already check this exists");
+    // The SymbolUses verifier which checks that this exists may not have been
```

Comment:
@lattner / @darthscsi / anyone who knows more about verifiers... Is this correct?

Reply:
Symbol use verifier is run before op verifier.

Reply:
Are you sure that hasn't changed recently? This change makes the same error message appear instead of asserting. The change in error messages in another test also implies the order in which verifiers are run has changed.

Reply:
Yeah, this commit changed that behavior: https://github.com/llvm/llvm-project/commit/5232c5c5d41489a07d0e72a0e87229a58950f94a